README.md

```diff
@@ -1,75 +1,75 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Hello Project
-
-> First microverse Project.
-
-
-## Built With
-
-- HTML
-- CSS
-- Web Development
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Project
+
+> First microverse Project.
+

```

Comment:
- [ ] Hey @Javed-Shahbaz you did well using the default readme template 👏. However, I believe you should update it to match your project. For example in this project, you don't have tests, you don't neeed to deploy you project... So, you can remove `line 35`, `line 37` , `line 14`, `line 16`, `line 21`, `line 22`...
- [ ] You are the only one working on this project, so I think you can delete the `Author 2` and update the `Author 1` with your `github`, `linkedin` and `twitter` links. 👌

### Note:
Having a well documented `readme` file is very important because it shows how to use your project.